confidant_client/__init__.py

```diff
@@ -455,53 +455,18 @@ def revert_credential(
         # Return a dict, always with an attribute that specifies whether or not
         # the function was able to successfully get a result.
         ret = {'result': False}
-        # Find the current revision
```

Comment:
Assuming we're removing all the client side checks because the server checks are more than enough?

Reply:
Well, mostly because we want to be able to do reverts without needing get permission. The server-side checks do also now do these checks as well, so it's safe to remove.